Model/AbstractOpsRegistry.php

```diff
@@ -79,7 +84,11 @@ public function stop($opName, $payload = [])
         }
 
         $opId = $this->getOpId();
-        $workTime = microtime(true) - $this->tsStart[$opId];
+        if (key_exists($opId, $this->tsStart)) {
```

Comment:
Not sure to understand why you changed it this way. Did you have any "invalid index" issue?